libr/core/cbin.c

```diff
@@ -1616,7 +1616,11 @@ static int bin_relocs(RCore *r, int mode, int va) {
 				}
 
 				// check if name is available
-				pj_ks (pj, "name", (relname && strcmp (relname, "")) ? relname : "N/A");
+				if (relname && *relname) {
+					pj_ks (pj, "name", relname);
+				} else {
+					pj_knull (pj, "name");
```

Comment:
what about not showing this field just in case there's no name, programatically it will be handled the same when parsing it and it is how its done in other places in r2

Reply:
It will not be handled the same in almost all parsers. But whether it is better in this case to omit the entire name or use null is a subjective question. If you prefer to have it removed, I can do that.